MediaBrowser.WebDashboard/dashboard-ui/strings/en-GB.json

```diff
@@ -700,8 +700,8 @@
   "LabelCompleted": "Completed",
   "LabelComponentsUpdated": "The following components have been installed or updated:",
   "LabelConfigureSettings": "Configure settings",
-  "LabelConnectGuestUserName": "Their Jellyfin Connect e-mail address or user name:",
```

Comment:
The meaning of the strings have changed here so any translations will be incorrect. The proper thing to do would be to create new strings and let the translations get updated in time. This also lets Sparky remove the unused connect strings.

Reply:
I already did that though, sort of...

Reply:
Whoops, you renamed them and I didn't notice! It might still be okay to leave the old strings in since Sparky said he was removing them soon.